frontend/src/metabase/dashboard/actions.js

```diff
@@ -1127,7 +1127,8 @@ export const fetchDashboardParameterValuesWithCache = createThunkAction(
 
       return {
         cacheKey,
-        results: results.map(result => [].concat(result)),
+        results: values.map(value => [].concat(value)),
```

Comment:
`values.flat()`? 🤔

Reply:
I think the desired result is an array of arrays (like `[["foo", "bar"], ["abc", "123"]]` because sometimes the second entry is used for remapping. The issue is that sometimes we receive `["foo", "bar"]` and need to normalize it to be doubly wrapped.